plugins/apitracing/src/lib/config/FunctionDefinitions.cpp

```diff
@@ -110,6 +110,38 @@ namespace ApiTracing
         return parameterInformationList;
     }
 
+    std::vector<ParameterInformation>
+    FunctionDefinitions::getStructParameterInformation(const YAML::Node& parameters, // NOLINT(misc-no-recursion)
+                                                       uint16_t addressWidth)
+    {
+        std::vector<ParameterInformation> parameterInformationList;
+
+        for (const auto parameter : parameters)
+        {
+            auto backingParameterName = parameter.first.as<std::string>();
+            auto parameterInformation = parameters[backingParameterName];
```

Comment:
This makes little sense since you already have an iterator that points to this value.